database.sql

```diff
@@ -0,0 +1,36 @@
+CREATE DATABASE family;
+
+USE family;
+
+CREATE TABLE `users` (
+	`id` INT(10) NOT NULL AUTO_INCREMENT ,
+	`name` VARCHAR(25) NOT NULL,
+	`status` VARCHAR(50) NOT NULL,
+	`age` INT(4) NOT NULL,
+	`address` VARCHAR(100) NOT NULL,
+	`approve_status` TINYINT (1) NOT NULL,
+	`password` VARCHAR(100) NOT NULL,
```

Comment:
It can have even more then 100 symbols. If i create password with 30 symbols and hash will have more than 100.